src/tinyformat.h

```diff
@@ -1061,6 +1061,14 @@ std::string format(const std::string &fmt, const Args&... args)
     return oss.str();
 }
 
+// This is to make the linter happy because it the variadic script/python variadic checker does not deal with
```

Comment:
Noticed a small typo
```suggestion
// This is to make the linter happy because the variadic script/python variadic checker does not deal with
```